src/components/templates/PartyMap/components/RoomAttendance.tsx

```diff
@@ -1,6 +1,6 @@
-import React, { useMemo } from "react";
+import React, { FC, useMemo } from "react";
```

Comment:
Prefer using `React.FC` directly where it's used, rather than importing it individually.

Reply:
Though given that's already the case, I think this import just isn't needed at all.